icu4c/source/i18n/calendar.cpp

```diff
@@ -1301,6 +1312,9 @@ int32_t Calendar::getRelatedYear(UErrorCode &status) const
             year -=5492; break;
         case CALTYPE_DANGI:
             year -= 2333; break;
+        case CALTYPE_TIBETAN:
```

Comment:
I have change Calendar::getRelatedYear and put those into sublcass in https://github.com/unicode-org/icu/pull/2199 (and later in https://github.com/unicode-org/icu/pull/2215 ). Please rebase and follow that.